platforms/core-execution/file-watching/src/integTest/groovy/org/gradle/internal/watch/EnableFileSystemWatchingIntegrationTest.groovy

```diff
@@ -132,16 +137,39 @@ class EnableFileSystemWatchingIntegrationTest extends AbstractFileSystemWatching
         watchMode << WatchMode.values()
     }
 
-    def "cannot define a custom build scope cache dir when watching is explicitly enabled"() {
-        buildFile << """
-        """
-        file("buildSrc/build.gradle") << """
-        """
+    def "cannot define a custom build scope cache dir when watching is explicitly enabled and cache dir is on another fs"() {
```

Comment:
🤔 Since we are changing the name of the test already, can we use this opportunity to change the wording from "cannot" to "fails when"? The user obviously "can", it's just that the build will fail.